starters/next-drupal-starter/__tests__/isMultiLanguage.test.js

```diff
@@ -0,0 +1,18 @@
+import { afterEach, describe, it, expect, vi } from "vitest";
+import * as utils from "../lib/isMultiLanguage";
+
+const spy = vi.spyOn(utils, "isMultiLanguage");
+
+describe("isMultiLanguage()", () => {
+  afterEach(() => {
+    vi.clearAllMocks();
+  });
+  it("should return true if there are multiple locales", () => {
+    expect(utils.isMultiLanguage(["en", "es"])).toBe(true);
+    expect(spy).toHaveBeenCalledTimes(1);
```

Comment:
Which is the value to do this?

Reply:
Which part, the spy?

Reply:
Yes, and the number of called times

Reply:
To make sure the code we want called was called. Maybe it doesn't have a lot of value here.

Reply:
I removed the spies. I think if we want to increase coverage we can bring them back to spy on `console.logs`, but I don't think I was using them very effectively as it was :)